src/plugins/event-plugins/FetchPlugin.ts

```diff
@@ -242,9 +242,10 @@ export class FetchPlugin extends MonkeyPatched<Window, 'fetch'> {
         response: Response
     ) => {
         if (this.config.recordAllRequests || !response.ok) {
+            const { status, statusText } = response;
             httpEvent.response = {
-                status: response.status,
-                statusText: response.statusText
```

Comment:
question: When `statusText` is undefined, does the key appear in the PutRumEvent payload? I believe `JSON.stringify` removes keys where the value is `undefined`.

Reply:
Yeah you're right, closing. A customer raised this issue on the backend, which is a separate domain.